ledger/controllers/wallet.js

```diff
@@ -379,28 +368,39 @@ const write = function (runtime, apiVersion) {
         surveyorIds: surveyorIds,
         altcurrency: wallet.altcurrency,
         probi: result.probi,
-        count: votes
+        count: totalVotes
       }
     }
     await viewings.update({ viewingId: viewingId }, state, { upsert: true })
 
     const picked = ['votes', 'probi', 'altcurrency']
     result = underscore.extend({ paymentStamp: now }, underscore.pick(result, picked))
-    if (apiVersion === 1) {
-      reply(underscore.omit(underscore.extend(result, {satoshis: Number(result.probi)}), ['probi', 'altcurrency']))
-    } else {
-      reply(result)
+
+    reply(result)
+
+    if (grantVotes > 0) {
+      await runtime.queue.send(debug, 'contribution-report', underscore.extend({
+        paymentId: paymentId,
+        address: wallet.addresses[result.altcurrency],
+        surveyorId: surveyorId,
+        viewingId: viewingId,
+        fee: grantFee,
+        votes: grantVotes,
+        cohort: 'grant'
```

Comment:
I think this should be grantCohort

Reply:
resolved in https://github.com/brave-intl/bat-ledger/pull/516/commits/f062d93b5091fe00ab2a04494ac47d7fccd32b44